src/test/TEST_bench/benchmark_ij.saved.tioga

```diff
@@ -84,19 +84,19 @@ Final Relative Residual Norm = 4.476621e-09
 
 # Output file: benchmark_ij.out.22
 Iterations = 13
-Final Relative Residual Norm = 1.703911e-09
+Final Relative Residual Norm = 1.810780e-09
```

Comment:
Residuals are different because the operator matrices A of BoomerAMG changed (because of CSR sorting taking place now). The differences correspond to cases using rocSPARSE's SpGEMM